audio_analysis_test.go

```diff
@@ -0,0 +1,142 @@
+package spotify
+
+import (
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+const FIELDS_DIFFER_TEMPLATE = "Actual response is not the same as expected response on field %s"
```

Comment:
The `ALL_CAPS` naming convention is not idiomatic Go.  Additionally, this constant need not be exported.

Rename this to `fieldsDifferTemplate`.